MonoGame.Framework.Content.Pipeline/Graphics/DxtTexture/Bitmap.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TextureSquish
+{
+    public class Bitmap

```

Comment:
I think we should remove the `public` off here if we decide to merge this... as we don't want to make this compressor a part of our public API.